activerecord/test/cases/associations/belongs_to_associations_test.rb

```diff
@@ -1354,6 +1354,15 @@ def test_polymorphic_with_custom_primary_key
     assert_equal toy, sponsor.reload.sponsorable
   end
 
+  def test_polymorphic_with_false
+    assert_nothing_raised do
+      Class.new(ActiveRecord::Base) do
+        def self.name; "Category"; end
+        belongs_to :category, polymorphic: false
```

Comment:
`belongs_to :category` on Category model is a bit weird.
Can you change the name to Post for example?

Reply:
done.